app/controllers/user_mentee_applications_controller.rb

```diff
@@ -24,6 +24,7 @@ def create
 
     @user_mentee_application = UserMenteeApplication.new(user_mentee_application_params)
     @user_mentee_application.user = current_user
+    @user_mentee_application.mentee_application_states.build(status: :pending)
```

Comment:
Would there ever be a situation where a `mentee_application` exists without having any `mentee_application_state`? I only ask because if this situation is impossible, then I might think it's a good idea to create the associated default `mentee_application_state` in an `after_create` callback. Not sure on this though, just something I'm curious about.

Reply:
I was trying not to use a callback but this approach is giving me test issues. I'll try the callback.

Reply:
Ok, after trying both, I think I lean more towards the callback approach.